### PR TITLE
Add a workaround for a flaky test

### PR DESCRIPTION
**Pull request by nunnatsa**
**What this PR does / why we need it**:

The `"console should reach kubevirt-plugin manifests"` functional test
became flaky. The reason for that may be a bug in the console code,
that sometimes return partial and corrupted json response.

This PR reads the json in a retry mechanism to give the console plugin
pod a chance to return a valid response.

**Jira Ticket**:

```jira-ticket
none
```

**Release note**:

```release-note
none
```

### CODE REVIEW

File: tests/func-tests/console_plugin_test.go [hunk 1]
```diff
@@ -65,29 +65,35 @@ var _ = Describe("kubevirt console plugin", Label(tests.OpenshiftLabel, "console
...
+			"name": "hyperconverged-cluster-operator",
+		}, client.InNamespace(tests.InstallNamespace))).To(Succeed())
+
+		Expect(hcoPods.Items).ToNot(BeEmpty())
+
+		testConsolePod := hcoPods.Items[0]
+		command := fmt.Sprintf(`curl -ks https://%s:%d/plugin-manifest.json`,
+			pluginServiceName, pluginServicePort)
+
+		Eventually(func(g Gomega, ctx context.Context) string {
```
Comment (avlitman) [inline tests/func-tests/console_plugin_test.go:79]: maybe let's add here a comment with the console bug number? that this Eventually can be removed once it's fixed.

**Comment by avlitman** [comment]
in the pr description:
> will be useful anyway

can you explain why? since we use Eventually now? is there additional reason?

File: tests/func-tests/console_plugin_test.go (hunk 1 above)
Comment (nunnatsa) [reply tests/func-tests/console_plugin_test.go:79]: I prefer to not add comments

**Comment by nunnatsa** [comment]
Closed in favor of #3242 3242

**Pull request merged**